sunpy/tests/pytest_intercept_remote/plugin.py

```diff
@@ -0,0 +1,91 @@
+import json
```

Comment:
Code cov seems to suggest none of this is covered. I assume that's an issue with codecov?

Reply:
Hey, @nabobalis, so I fixed the issue with code cov but it seems `sunpy/tests/` is omitted in coverage reports, could you suggest a solution or perhaps a better location for the plugin?

Reply:
Hey @nabobalis, so I fixed the coverage issue and tested the plugin locally; it seems code cov is not so happy with the plugin's location, since `sunpy/tests/` is omitted by coverage reports.

Could you suggest some solution or perhaps a better location for the plugin?

Reply:
What if we remove the code that prevents codecov from measuring that location?

Reply:
yes we can
but isn't omitting `*/tests/*`  essential; I'm not sure how would we particularly select `sunpy/tests/` and omit others?

Reply:
Essential? I don't see why.

Reply:
correct me if I am wrong, I thought test files popping in coverage reports is undesirable.

nvm, I think replacing `sunpy/*/tests/*` by `sunpy/*/tests/!(pytest_intercept_remote)` should include the plugin in coverage and exclude other test folders.

Reply:
Not sure it really matters.

If that works, thats great.

Reply:
considering the behaviour of codecov, I threw the plugin into a separate package at <a href="https://github.com/devanshshukla99/pytest-intercept-remote">pytest-intercept-remote</a> and everything seems fine there.

Would you prefer this to be within sunpy or as a separate package?
personally, I feel like keeping this separate is better considering the changes required in coverage options and for testing the plugin.

Reply:
Yeah I think making it seperate is a better idea.

Reply:
So, should I close this and open a new one or make the required changes in this pr only?

Reply:
Make the changes here.